modules/os2dagsorden/os2dagsorden_settings/os2dagsorden_settings.module

```diff
@@ -186,6 +189,7 @@ function os2dagsorden_settings_export_to_file() {
     $setting_array['os2dagsorden_pws_sofd_group']['os2dagsorden_pws_v_acadregroups_url'] = variable_get('os2dagsorden_pws_v_acadregroups_url');
   }
 
+  // @CODEREVIEW: Shoudn't the default texts (e.g. "Her vises de kommende 5 møder i …") be translatable?
```

Comment:
I don't see a point in adding it to the translation.
Currently Dagsorden is single language application. And those values are already changeable by the settings in os2dagsorden_settings.admin.inc.